MelonLoader.Installer/MLManager.cs

```diff
@@ -49,9 +50,22 @@ public static async Task<bool> Init()
             return true;
 
         inited = await RefreshVersions();
+
         return inited;
     }
 
+    public static void OpenSteamGameProperties(string? appId)
```

Comment:
This class is meant for MelonLoader-related operations only. Move this to the SteamLauncher class